sdk/storage/Azure.Storage.Blobs.Batch/src/Models/Internal/DeleteSnapshotsOptionType.cs

```diff
@@ -8,11 +8,11 @@ internal enum DeleteSnapshotsOptionType
         /// <summary>
         /// Include.
         /// </summary>
-        Include,
+        Include=1,
```

Comment:
Why is designating the value needed?

Reply:
Oh I see because if not, one of these values would get defaulted to 0, let me see if there's a better way since it's not the norm to be setting values for our enums.

Reply:
Actually I think  values is fine, I think we probably need a default value though. I am kinda nervous about this because if a customer was setting the exact value (e.g. `batch.DeleteBlob(blobs[0].Uri, 1), then it's a breaking change cause they weren't expecting the value to change.
@seanmcc-msft  what do you think?

Reply:
![image](https://user-images.githubusercontent.com/61715331/125108097-6807f400-e096-11eb-9e43-dd6c70e63251.png)